resources/views/components/form.blade.php

```diff
@@ -1,4 +1,4 @@
-<form method="{{ (isset($method) && $method === 'GET') ? 'GET' : 'POST' }}" action="{{ is_array($route) ? route(...$route) : route($route) }}">
+<form method="{{ (isset($method) && $method === 'GET') ? 'GET' : 'POST' }}" action="{{ is_array($route) ? route(...$route) : route($route) }}"{{ (!(isset($method) && $method === 'GET') && isset($enctype)) ? ('enctype="' . $enctype . '"') : ''}}>
```

Comment:
Das muess mer na ändere, damit de enctype richtig isch. (Susch het mer nacher double double quotes ""enctype"")

```suggestion
<form method="{{ (isset($method) && $method === 'GET') ? 'GET' : 'POST' }}" action="{{ is_array($route) ? route(...$route) : route($route) }}"{{ (!(isset($method) && $method === 'GET') && isset($enctype)) ? ('enctype=' . $enctype) : ''}}>
```

Reply:
Tatsächlich, enctype=multipart/form-data ganz ohni quotes isch korrekt. HTML isch scho öppis komischs.